pandas/tseries/tdi.py

```diff
@@ -440,21 +439,18 @@ def astype(self, dtype):
 
         if dtype == np.object_:
             return self.asobject
-        elif dtype == _INT64_DTYPE:
-            return self.asi8.copy()
         elif dtype == _TD_DTYPE:
```

Comment:
similar changes here:

``is_timedelta64_dtype(dtype)`` for ``dtype == _TD_DTYPE``